baseplate/clients/memcache/__init__.py

```diff
@@ -125,135 +148,148 @@ class MonitoredMemcacheConnection:
             return self.pooled_client.set_many(values, expire=expire, noreply=noreply)
 
-    def replace(self, key, value, expire=0, noreply=None):
+    def replace(
+        self, key: Key, value: Any, expire: int = 0, noreply: Optional[bool] = None
+    ) -> bool:
         with self._make_span("replace") as span:
             span.set_tag("key", key)
             span.set_tag("expire", expire)
             span.set_tag("noreply", noreply)
             return self.pooled_client.replace(key, value, expire=expire, noreply=noreply)
 
-    def append(self, key, value, expire=0, noreply=None):
+    def append(self, key: Key, value: Any, expire: int = 0, noreply: Optional[bool] = None) -> bool:
         with self._make_span("append") as span:
             span.set_tag("key", key)
             span.set_tag("expire", expire)
             span.set_tag("noreply", noreply)
             return self.pooled_client.append(key, value, expire=expire, noreply=noreply)
 
-    def prepend(self, key, value, expire=0, noreply=None):
+    def prepend(
+        self, key: Key, value: Any, expire: int = 0, noreply: Optional[bool] = None
+    ) -> bool:
         with self._make_span("prepend") as span:
             span.set_tag("key", key)
             span.set_tag("expire", expire)
             span.set_tag("noreply", noreply)
             return self.pooled_client.prepend(key, value, expire=expire, noreply=noreply)
 
-    def cas(self, key, value, cas, expire=0, noreply=None):
+    def cas(
+        self, key: Key, value: Any, cas: int, expire: int = 0, noreply: Optional[bool] = None
+    ) -> Optional[bool]:
         with self._make_span("cas") as span:
             span.set_tag("key", key)
             span.set_tag("cas", cas)
             span.set_tag("expire", expire)
             span.set_tag("noreply", noreply)
             return self.pooled_client.cas(key, value, cas, expire=expire, noreply=noreply)
 
-    def get(self, key, **kwargs):
+    def get(self, key: Key, default: Any = None) -> Any:
         with self._make_span("get") as span:
             span.set_tag("key", key)
+            kwargs = {}
+            if default is not None:
+                kwargs["default"] = default
             return self.pooled_client.get(key, **kwargs)
 
-    def get_many(self, keys):
+    def get_many(self, keys: Sequence[Key]) -> Dict[Key, Any]:
         with self._make_span("get_many") as span:
             span.set_tag("key_count", len(keys))
             span.set_tag("keys", make_keys_str(keys))
             return self.pooled_client.get_many(keys)
 
-    def gets(self, key, **kwargs):
+    def gets(
+        self, key: Key, default: Optional[Any] = None, cas_default: Optional[Any] = None
+    ) -> Tuple[Any, Any]:
         with self._make_span("gets") as span:
             span.set_tag("key", key)
-            return self.pooled_client.gets(key, **kwargs)
+            return self.pooled_client.gets(key, default=default, cas_default=cas_default)
```

Comment:
I might need to change this to look like `get()`'s internals on L215? I think this was done because those params don't exist in some versions of redis-py?